wgpu-hal/src/gles/adapter.rs

```diff
@@ -318,6 +358,12 @@ impl super::Adapter {
             ver >= (3, 1),
         );
 
+        #[cfg(not(target_arch = "wasm32"))]
+        private_caps.set(super::PrivateCapabilities::INDEX_BUFFER_ROLE_CHANGE, true);
+
+        #[cfg(target_arch = "wasm32")]
+        private_caps.set(super::PrivateCapabilities::EMULATE_BUFFER_MAP, true);
```

Comment:
This is not a capability. It either needs to be reversed (e.g. `NATIVE_MAPPING`), or moved to `workarounds`